llvm/lib/MC/MCWin64EH.cpp

```diff
@@ -1402,6 +1402,9 @@ static void ARM64EmitUnwindInfo(MCStreamer &streamer, WinEH::FrameInfo *info,
     // here, but we'd have to emit the pdata, the xdata header, and the
     // epilogue scopes later, since they depend on whether the we need to
     // split the unwind data.
+    //
+    // If this is fixed, remove code in AArch64ISelLowering.cpp that
```

Comment:
What's "this" here and what would "fixed" mean. Maybe it needs a "see <link>". As out of the context of this PR it doesn't make much sense.

Reply:
This is a large comment block, that starts out with:
```
    // FIXME: GetAbsDifference tries to compute the length of the function
    // immediately, before the whole file is emitted, but in general
    // that's impossible: the size in bytes of certain assembler directives
    // like .align and .fill is not known until the whole file is parsed and
    // relaxations are applied. Currently, GetAbsDifference fails with a fatal
    // error in that case.
```
(And then goes on about potential ways of fixing it.) So this comment says that if this whole general issue/limitation is fixed, we can remove this tweak.